### PR TITLE
Format PoCo2 solidity files & update copyright notices

### PR DESCRIPTION
**Pull request by zguesmi**
Manual update:
* [Remove warning "Only state variables can have a docstring. This will be disallowed in 0.7.0"](https://github.com/iExecBlockchainComputing/PoCo/commit/00d020ea6042193f7b87893cc4a9e796c7c8c6c9)

### CODE REVIEW

File: contracts/modules/interfaces/IexecPoco2.sol
Comment (zguesmi) [inline contracts/modules/interfaces/IexecPoco2.sol:1]: image

File: contracts/modules/delegates/IexecPoco2Delegate.sol
Comment (zguesmi) [inline contracts/modules/delegates/IexecPoco2Delegate.sol:1]: image

File: contracts/modules/delegates/IexecPoco2Delegate.sol [hunk 1]
```diff
@@ -25,574 +10,541 @@ import "./SignatureVerifier.sol";
...
+            reward(deal.dataset.owner, deal.dataset.price, _taskid);
+        }
+        // unlock pool stake
+        unlock(deal.workerpool.owner, poolstake);
+        // pool reward performed by consensus manager
+
+        // Retrieve part of the kitty
+        uint256 kitty = m_frozens[KITTY_ADDRESS];
+        if (kitty > 0) {
+            kitty = kitty.percentage(KITTY_RATIO).max(KITTY_MIN).min(kitty); // fraction // at least this // but not more than available
```
Comment (jeremyjams) [inline contracts/modules/delegates/IexecPoco2Delegate.sol:40]: Could you please update this line (note: code will soon change though)?

File: contracts/modules/delegates/IexecPoco2Delegate.sol [hunk 2]
```diff
@@ -25,574 +10,541 @@ import "./SignatureVerifier.sol";
...
+        emit TaskConsensus(_taskid, resultHash);
+        emit TaskReveal(_taskid, _msgSender(), _resultDigest);
+        emit TaskFinalize(_taskid, _results);
+
+        executeCallback(_taskid, _resultsCallback);
+    }
+
+    function reveal(
+        bytes32 _taskid,
+        bytes32 _resultDigest // worker
```
Comment (jeremyjams) [inline contracts/modules/delegates/IexecPoco2Delegate.sol:249]: Can you move comment?

File: contracts/modules/delegates/IexecPoco2Delegate.sol (hunk 1 above)
Comment (zguesmi) [reply contracts/modules/delegates/IexecPoco2Delegate.sol]: Done
[Fix comment](https://github.com/iExecBlockchainComputing/PoCo/pull/54/commits/7d5db24762238c322dc6755cfa52036dc80d956a)
Thanks.

File: contracts/modules/delegates/IexecPoco2Delegate.sol (hunk 2 above)
Comment (zguesmi) [reply contracts/modules/delegates/IexecPoco2Delegate.sol]: Deleted
[Fix comment](https://github.com/iExecBlockchainComputing/PoCo/pull/54/commits/f2ac8eab73bcbb26d95ba0387bf715fc579eb28d)

File: contracts/modules/delegates/IexecPoco2Delegate.sol (hunk 1 above)
Comment (zguesmi) [reply contracts/modules/delegates/IexecPoco2Delegate.sol]: How about this?
[Fix kitty comment](https://github.com/iExecBlockchainComputing/PoCo/pull/54/commits/3915c9bb77cb9514803502797a7183c46ec7f30a)

**Review by jeremyjams — APPROVED** [review]
Thanks :ok_hand:

**Pull request merged**